botogram/frozenbot.py

```diff
@@ -232,6 +232,11 @@ def available_commands(self, all=False):
             if all or not is_hidden:
                 yield command
 
+    def sorted_available_commands(self, all=False):
```

Comment:
Why is this a different function? You should be able to add this to `available_commands`.